### PR TITLE
DOCSUP-7099: edit and translate (runningConcurrency, max_parallel_replicas)

### PR DESCRIPTION
**Pull request by lehasm**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:
- Documentation for #18508 and #21113

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -1097,14 +1097,25 @@ See the section “WITH TOTALS modifier”.
...
+Possible values:
+
+-   Positive integer.
+
+Default value: `1`.
 
-In addition, this setting will produce incorrect results when joins or subqueries are involved, and all tables don't meet certain conditions. See [Distributed Subqueries and max_parallel_replicas](../../sql-reference/operators/in.md#max_parallel_replica-subqueries) for more details.
+**Additional Info** 
+
+This setting is only useful for replicated tables with a sampling key. A query may be executed faster by executing on several servers in parallel. But query performance may degrade in some cases:
```
Comment (olgarev) [inline docs/en/operations/settings/settings.md:1110]:
```suggestion
This setting is only useful for replicated tables with a sampling key. A query may be processed faster if it is executed on several servers in parallel. But query performance may degrade in some cases:
```

File: docs/en/sql-reference/functions/other-functions.md [hunk 2]
```diff
@@ -911,29 +911,30 @@ Same as for [runningDifference](../../sql-reference/functions/other-functions.md
...
 
 **Syntax**
 
 ``` sql
-runningConcurrency(begin, end)
+runningConcurrency(start, end)
 ```
 
-**Arguments**
+An event has start time and end time. Columns with these times must have the same data type. The start time is included in an event, while the stop time is excluded. The function analyses each event start time and calculates the total number of events that are active including the starting event.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/other-functions.md:922]:
```suggestion
Each event has a start time and an end time. Columns with these two values must be of the same data type. The start time is included in the event, while the end time is excluded. The function calculates the total number of events that are active at the specified start time.
```

File: docs/en/sql-reference/functions/other-functions.md [hunk 3]
```diff
@@ -911,29 +911,30 @@ Same as for [runningDifference](../../sql-reference/functions/other-functions.md
...
--   `begin` — A column for the beginning time of events (inclusive). [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
--   `end` — A column for the ending time of events (exclusive).  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
+!!! warning "Warning"
+    Events must be ordered by the start time in ascending order. If this requirement is violated the function raises an exception.
 
-Note that two columns `begin` and `end` must have the same type.
+**Arguments**
+
+-   `start` — A column with start time of events. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
+-   `end` — A column with end time of events.  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/other-functions.md:933]:
```suggestion
-   `start` — A column with the start time of events. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
-   `end` — A column with the end time of events.  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
```

File: docs/en/sql-reference/functions/other-functions.md [hunk 4]
```diff
@@ -942,31 +943,29 @@ Type: [UInt32](../../sql-reference/data-types/int-uint.md)
 Input table:
```
Comment (olgarev) [inline docs/en/sql-reference/functions/other-functions.md:943]:
```suggestion
Consider the table:
```

File: docs/ru/operations/settings/settings.md [hunk 5]
```diff
@@ -1087,8 +1087,22 @@ load_balancing = round_robin
...
-Для консистентности (чтобы получить разные части одного и того же разбиения), эта опция работает только при заданном ключе сэмплирования.
-Отставание реплик не контролируется.
+
+Возможные значения:
+
+-   Целое положительное число.
+
+**Дополнительная информация**
+
+Эта настройка полезна для реплицируемых таблиц с ключом семплирования. Запрос может обрабатываться быстрее, если он выполняется на нескольких серверах параллельно. Однако производительность обработки запроса может наоборот упасть в некоторых ситуациях:
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:1097]:
```suggestion
Эта настройка полезна для реплицируемых таблиц с ключом сэмплирования. Запрос может обрабатываться быстрее, если он выполняется на нескольких серверах параллельно. Однако производительность обработки запроса может наоборот упасть в некоторых ситуациях:
```

File: docs/ru/operations/settings/settings.md [hunk 6]
```diff
@@ -1087,8 +1087,22 @@ load_balancing = round_robin
...
+
+- Позиция ключа семплирования в ключе разбиения не позволяет выполнять эффективное сканирование.
+- Добавление ключа семплирования в таблицу делает фильтрацию по другим столбцам менее эффективной.
+- Ключ семплирования является выражением, которое сложно вычисляется.
+- У распределения задержек в кластере длинный хвост, из-за чего запросы к нескольким серверам увеличивают общее время задержки.
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:1102]:
```suggestion
- Позиция ключа сэмплирования в ключе разбиения не позволяет выполнять эффективное сканирование.
- Добавление ключа сэмплирования в таблицу делает фильтрацию по другим столбцам менее эффективной.
- Ключ сэмплирования является выражением, которое сложно вычисляется.
- У распределения задержек в кластере длинный хвост, из-за чего запросы к нескольким серверам увеличивают общее время задержки.
```

File: docs/ru/operations/settings/settings.md [hunk 7]
```diff
@@ -1087,8 +1087,22 @@ load_balancing = round_robin
...
+
+!!! warning "Предупреждение"
+    Параллельное выполнение запроса приведет к неверному результату, когда в запросе есть объединение или подзапросы и при этом таблицы не удовлетворяют некоторым требованиям. Подробности смотрите в [Распределенные подзапросы и max_parallel_replicas](../../sql-reference/operators/in.md#max_parallel_replica-subqueries).
```
Comment (olgarev) [inline docs/ru/operations/settings/settings.md:1105]:
```suggestion
    Параллельное выполнение запроса может привести к неверному результату, если в запросе есть объединение или подзапросы, и при этом таблицы не удовлетворяют некоторым требованиям. Подробности смотрите в разделе [Распределенные подзапросы и max_parallel_replicas](../../sql-reference/operators/in.md#max_parallel_replica-subqueries).
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 8]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
 
 То же, что и \[runningDifference\] (./other_functions.md # other_functions-runningdifference), но в первой строке возвращается значение первой строки, а не ноль.
 
+
+## runningConcurrency {#runningconcurrency}
+
+Определяет количество одновременно идущих событий в моменты начала событий.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:858]:
```suggestion
Определяет все моменты начала событий в указанном интервале и подсчитывает количество событий, происходящих одновременно в каждый из этих моментов. 
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 9]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+
+Определяет количество одновременно идущих событий в моменты начала событий. 
+
+**Синтаксис**
+
+``` sql
+runningConcurrency(start, end)
+```
+
+У каждого события есть время начала и время завершения. Столбцы с этими значениями должны содержать данные одинакового типа. Время начала включается в событие, а время завершения исключается из события. Для момента начала каждого события функция вычисляет количество идущих событий.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:866]:
```suggestion

```

File: docs/en/sql-reference/functions/other-functions.md [hunk 10]
```diff
@@ -911,62 +911,59 @@ Same as for [runningDifference](../../sql-reference/functions/other-functions.md
...
 
 **Syntax**
 
 ``` sql
-runningConcurrency(begin, end)
+runningConcurrency(start, end)
 ```
 
-**Arguments**
+Each event has a start time and an end time. Columns with these two values must be of the same data type. The start time is included in the event, while the end time is excluded. The function calculates the total number of active events for each start time in the specified period.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/other-functions.md:922]:
```suggestion

```

File: docs/en/sql-reference/functions/other-functions.md [hunk 11]
```diff
@@ -911,62 +911,59 @@ Same as for [runningDifference](../../sql-reference/functions/other-functions.md
 
 ## runningConcurrency {#runningconcurrency}
 
-Given a series of beginning time and ending time of events, this function calculates concurrency of the events at each of the data point, that is, the beginning time.
-
-!!! warning "Warning"
-    Events spanning multiple data blocks will not be processed correctly. The function resets its state for each new data block.
-
-The result of the function depends on the order of data in the block. It assumes the beginning time is sorted in ascending order.
+Calculates the number of events that are concurrent at event start time.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/other-functions.md:914]:
```suggestion
Each event has a start time and an end time. The start time is included in the event, while the end time is excluded. 

The function calculates the total number of active (concurrent) events for each start time in the specified period.

Columns with a start time and an end time must be of the same data type. 
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 12]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+
+!!! warning "Предупреждение"
+    Каждый блок данных обрабатывается независимо. Если события из разных блоков данных накладываются, то они не могут быть корректно обработаны.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:869]:
```suggestion
    Каждый кусок данных обрабатывается независимо. Если события из разных кусков данных накладываются по времени, они не могут быть корректно обработаны.
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 13]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+    События должны быть отсортированы по возрастанию времени начала. Если это требование нарушено, то функция вызывает исключение.
+
+**Аргументы**
+
+-   `begin` — Столбец с временем начала событий. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
+-   `end` — Столбец с временем завершения событий.  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:875]:
```suggestion
-   `end` — Столбец с временем окончания событий.  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md) или [DateTime64](../../sql-reference/data-types/datetime64.md).
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 14]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+
+У каждого события есть время начала и время завершения. Столбцы с этими значениями должны содержать данные одинакового типа. Время начала включается в событие, а время завершения исключается из события. Для момента начала каждого события функция вычисляет количество идущих событий.
+
+!!! warning "Предупреждение"
+    Каждый блок данных обрабатывается независимо. Если события из разных блоков данных накладываются, то они не могут быть корректно обработаны.
+    События должны быть отсортированы по возрастанию времени начала. Если это требование нарушено, то функция вызывает исключение.
+
+**Аргументы**
+
+-   `begin` — Столбец с временем начала событий. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:874]:
```suggestion
-   `begin` — Столбец с временем начала событий. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md) или [DateTime64](../../sql-reference/data-types/datetime64.md).
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 15]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+-   `end` — Столбец с временем завершения событий.  [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md), or [DateTime64](../../sql-reference/data-types/datetime64.md).
+
+**Возвращаемое значение**
+
+-   Количество одновременно идущих событий в момента начала каждого события.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:879]:
```suggestion
-   Количество одновременно идущих событий на каждый момент начала события в указанном интервале.
```

File: docs/ru/sql-reference/functions/other-functions.md [hunk 16]
```diff
@@ -852,6 +852,64 @@ WHERE diff != 1
...
+
+Тип: [UInt32](../../sql-reference/data-types/int-uint.md)
+
+**Пример**
+
+Для таблицы:
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/other-functions.md:885]:
```suggestion
Рассмотрим таблицу:
```

**Comment by lehasm** [comment]
There are some conflicts. I will try to resolve them a bit later.

**Comment by den-crane** [comment]
I see it removes these sentences

**In limited circumstances, this can make a query faster by executing it on more servers. ... There are cases where performance will not improve or even worsen:**

It's actually quite important. max_parallel_replicas is mostly unusable and in 99% cases makes performance worse.

**Comment by lehasm** [comment]
> In limited circumstances, this can make a query faster by executing it on more servers. ... There are cases where performance will not improve or even worsen:
It changed to _A query may be processed faster if it is executed on several servers in parallel. But the query performance may degrade in some cases:_ in _Additional Info_ section.
@den-crane , is that ok?

**Comment by den-crane** [comment]
@lehasm I have a concern about 'some cases', because it's like 99.999% cases.

**Comment by lehasm** [comment]
We can change it to^
_A query may be processed faster if it is executed on several servers in parallel. But there are *a lot of cases* when query performance may degrade:_
@den-crane, do you agree?

**Comment by lehasm** [comment]
Changed _some cases_ to _following cases_ (neutral variant in my opinion).

**Comment by den-crane** [comment]
ok

**Review by nikitamikhaylov — APPROVED** [review]

**Pull request merged**